onelogin_rules/onelogin_brute_force_by_ip.yml

```diff
@@ -0,0 +1,30 @@
+AnalysisType: rule
+Filename: onelogin_brute_force_by_ip.py
+RuleID: OneLogin.BruteForceByIP
+DisplayName: OneLogin Brute Force IP
+Enabled: true
+LogTypes:
+  - OneLogin.Events
+Tags:
+  - OneLogin
+Severity: Medium
+Reports:
+  MITRE ATT&CK:
+    - Brute Force:Password Spraying
+Description: A single ip address was denied access to OneLogin more times than the configured threshold.
+Threshold: 10
+DedupPeriodMinutes: 10
+Reference: https://developers.onelogin.com/api-docs/1/events/event-resource
+Runbook: Analyze the IP they came from, and other actions taken before/after.
+Tests:
+  -
+    Name: Normal Login Event
+    ExpectedResult: false
+    Log:
+      {
+        'event_type_id': 8,
+        'actor_user_id': 123456,
+        'actor_user_name': 'Bob Cat',
+        'user_id': 123456,
+        'user_name': 'Bob Cat'
+      }
```

Comment:
For threshold rules, the unit tests assume that the threshold would always be breached by a `True` return value.

So in this case, we could add a unit test with `'event_type_id': 6,` and `ExpectedResult: true` and it should work.

This comment applies to a few threshold rules in this PR.